### PR TITLE
Remove CircleCI configuration

### PR DESCRIPTION
**Pull request by blag**
CircleCI has been deactivated in the repository settings. This PR removes CircleCI configuration.

Note that Travis integration is still activated and configured.

### CODE REVIEW

**Comment by blag** [comment]
I'm not sure why GitHub is trying to run CircleCI test for this PR, since that webhook was disabled before I opened this PR.

I've reactivated the CircleCI webhook. Closing and reopening this PR to get it to pass CI checks. Will disable _after_ merging this PR.

**Comment by blag** [comment]
Whelp, that didn't work like I expected. 😆

Pushing an unrelated commit to kick CI.

**Comment by blag** [comment]
Merging, even though CircleCI webhooks were still fired even after being deactivated. I think this is a GitHub bug.

Hopefully future PRs will not cause CircleCI webhooks to be fired off.

**Comment by blag** [comment]
Closing. Reopening another PR.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # Deprecated
```
Comment (armab) [inline README.md]: I think we can freely remove `Deprecated` tag.

The fact that st2chatops relies on st2client.js makes all the difference.

**Pull request closed without merging**